packages/main/src/webComponents/Table/Table.mdx

```diff
@@ -58,27 +67,83 @@ const GrowingTable = () => {
   };
   return (
     <div style={{ height: '250px', overflow: 'auto' }}>
-      <Table
-        onLoadMore={onLoadMore}
-        growing={TableGrowingMode.Scroll}
-        headerRow={
-          <TableHeaderRow>
-            <TableHeaderCell>
-              <Label>Column 1</Label>
-            </TableHeaderCell>
-            <TableHeaderCell>
-              <Label>Column 2</Label>
-            </TableHeaderCell>
-          </>
-        }
-      >
+      <Table {...args} features={<TableGrowing onLoadMore={onLoadMore} type={TableGrowingMode.Scroll} />}>
         {rows}
       </Table>
     </div>
   );
 };
 ```
 
+## Table with row selection
+
+`Table` with selection feature (`<TableSelection mode="Multiple" />`):
+
+<Canvas of={ComponentStories.WithSelection} />
+
+<details>
+
+    <summary>Show code</summary>
+
+```jsx
+function TableWithRowSelection() {
+  const [mode, setMode] = useState(TableSelectionMode.Multiple);
+  return (
+    <>
+      <SegmentedButton
+        onSelectionChange={(e) => {
+          setMode(e.detail.selectedItems[0].textContent);
+        }}
+      >
+        {Object.values(TableSelectionMode).map((selectionMode) => (
+          <SegmentedButtonItem key={selectionMode} selected={selectionMode === mode}>
+            {selectionMode}
+          </SegmentedButtonItem>
+        ))}
+      </SegmentedButton>
+      <Table {...args} features={<TableSelection mode={mode} />}>
```

Comment:
I would add the `headerRow` in this example for the sake of completeness. Users who a not familiar with storybook might be confused what `...args` should be